bundles/org.openhab.binding.philipsair/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,185 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="philipsair"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="pwr">
+		<item-type>Switch</item-type>
+		<label>Power</label>
+		<description>Device power</description>
+	</channel-type>
+	<channel-type id="om">
+		<item-type>String</item-type>
+		<label>Fan Speed</label>
+		<description>Fan speed</description>
+		<state pattern="%s">
+			<options>
+				<option value="s">Silent</option>
+				<option value="1">1</option>
+				<option value="2">2</option>
+				<option value="3">3</option>
+				<option value="t">Turbo</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="mode">
+		<item-type>String</item-type>
+		<label>Mode</label>
+		<description>Filter mode</description>
+		<state pattern="%s">
+			<options>
+				<option value="P">Auto</option>
+				<option value="A">Allergen</option>
+				<option value="S">Sleep</option>
+				<option value="M">Manual</option>
+				<option value="B">Bacteria</option>
+				<option value="N">Night</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="uil">
+		<item-type>Switch</item-type>
+		<label>Buttons</label>
+		<description>Buttons light</description>
+		<state pattern="%s"/>
+	</channel-type>
+	<channel-type id="ddp">
+		<item-type>String</item-type>
+		<label>Displayed index</label>

```

Comment:
The first letter of every word in a label should be capitalized.
```suggestion
		<label>Displayed Index</label>
```

Reply:
Nice spot, an obvious issues - sorry ;)